src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .map((style) => style.split(':'))
```

Comment:
[CODE KNOWLEDGE] - There's no need to split the style into key and value and then reassign them in the reduce function. You can trim the key and value directly in the map function before reducing them into an object.